instrumentation/apache-pulsar/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pulsar/PulsarTelemetry.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.pulsar;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.propagation.TextMapPropagator;
+
+public final class PulsarTelemetry {
+
+  private static final String INSTRUMENTATION = "io.opentelemetry:pulsar-client";
+  public static final AttributeKey<String> TOPIC = AttributeKey.stringKey("messaging_pulsar_topic");
```

Comment:
By default instrumentations in this repository fill only attributes described in opentelemetry semantic conventions.  Attributes that are not in semantic conventions are only added when they are explicitly enable with a flag like `-Dotel.instrumentation.pulsar.experimental-span-attributes=true`